pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -65,6 +72,8 @@ pub async fn download_layer_file<'a>(
         layer_metadata.generation,
     );
 
+    tracing::info!("download_layer_file: {remote_path} -> {local_path}");
```

Comment:
Is this log message at info! level intentional? At least for attached-mode tenants, it's probably redundant to the logging that struct Layer does.